client/src/components/UpcomingTripsCard/UpcomingTripsCard.css

```diff
@@ -0,0 +1,60 @@
+.UpcomingTripsCard {
+    font-family: var(--default-font);
+    display: flex;
+    flex-direction: column;
+}
+
+header {
+    font-size: 2.25rem;
+    text-align: center;
+    color: var(--white);
+}
```

Comment:
This will clash with other CSS! Can you namespace this under the `.UpcomingTripsCard` class?